dist/main.js

```diff
@@ -0,0 +1,33 @@
+const model = new Model()
+const render = new Renderer()
+  
+const loadPage = async function(){
+    await model.getDataFromDB()
+    render.renderDbData(model.cityData)
+}
+
+loadPage()
+
+const handleSearch = async function(){
+    const cityName = $('#cityName').val()
+    await model.getCityData(cityName)
+    render.renderData(model.cityData[model.cityData.length - 1])
+    model.cityData.splice([model.cityData.length - 1], 1)
+}
+
+$('.weather').on('click', '.delete', async function(){
+    const cityName =  $(this).closest('.oldCity').find('.cityName').text()
+    model.removeCity(cityName)
+    $(this).closest('.oldCity').remove()
```

Comment:
Why is the controller removing something from the DOM. This is the renderers job.